disassembler.cc

```diff
@@ -77,7 +77,7 @@ int registerIndexForString(const char *str)
 		}
 		return idx + 64;
 	}
-	if (str[0] == 'f')
+	if (str[0] == 'f' && str[1] != 'p')
```

Comment:
Can you just push these separately (they're simple and obvious bug fixes - it would be nice to have tests for them though...)